### ISSUE TITLE
doc_markdown false positive for comments inside docs test

### ISSUE THREAD

**Issue by zetok** [Manishearth/rust-clippy#1073]
Or at least I *think* that this is the case.

`clippy 0.0.78`

[False positive](https://travis-ci.org/zetok/tox/jobs/143272819#L2258):
```
/home/travis/build/zetok/tox/src/toxcore/state_format/old.rs:366:27: 366:37 warning: you should put `PackedNode` between ticks in the documentation, #[warn(doc_markdown)] on by default

/home/travis/build/zetok/tox/src/toxcore/state_format/old.rs:366         // here would go `PackedNode`s, but since their length is `0`..
```

Code in question; `PackedNode` is a struct:
https://github.com/zetok/tox/blob/0c3967babffe5263c3795cff18494987ed624325/src/toxcore/state_format/old.rs#L366

**Comment by mcarton** [comment]
Thanks for the report!
Should be fixed in the next release.
